client/components/SocialClub/SocialClubPage.js

```diff
@@ -67,18 +61,19 @@ class SocialClubPage extends Component {
     });
   }
 
-  join(details={'club_id': 3, 'email': 'ig@uk.com'}){
+  join(){
```

Comment:
Let's use a more descriptive name here, `join` what?
Maybe `JoinClub` will be more appropriate here, what do you think?

Reply:
Ok... noted